src/store/bitpay-id/bitpay-id.effects.ts

```diff
@@ -169,6 +169,8 @@ export const startLogin =
         LogActions.info('Successfully authenticated BitPayID credentials.'),
       );
 
+      dispatch(logSegmentEvent('track', 'Log In User success', {}, true));
```

Comment:
May want to differentiate the different types of login, this will only track basic login success. eg. 2fa authenticator login, 2fa email login

Reply:
I added the different types of successful login. Thanks.